prometheus_client/bridge/newrelic.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import absolute_import
+
+import newrelic.agent
+
+from .. import core
+
+
+@newrelic.agent.data_source_factory(name='Prometheus')
+class PrometheusDataSource(object):
+    def __init__(self, settings, environ):
+        self.registry = settings.get('registry', core.REGISTRY)
+        self.prefix = settings.get('prefix', '')
+
+    def __call__(self):
+        prefixstr = ''
+        if self.prefix:
+            prefixstr = self.prefix + '.'
+
+        for metric in self.registry.collect():
+            for name, labels, value in metric.samples:
```

Comment:
Yep, I did see [this bit of the docs](https://docs.newrelic.com/docs/agents/python-agent/supported-features/python-custom-metrics#pre-aggregated-metrics) about returning a dictionary, but for some reason didn't think to apply it here. I'll take a look.